src/Entities/Entity.php

```diff
@@ -9,53 +9,20 @@
 
 namespace SphereMall\MS\Entities;
 
+use SphereMall\MS\Lib\BaseModel;
 use SphereMall\MS\Lib\Helpers\ClassReflectionHelper;
-use SphereMall\MS\Lib\Traits\InteractsWithProperties;
 
 /**
  * Class Entity
  * @package SphereMall\MS\Entities
  */
-class Entity
+class Entity extends BaseModel
 {
-    use InteractsWithProperties;
-
-    #region [Constructor]
-
-    /**
-     * Entity constructor.
-     * @param array $data
-     */
-    public function __construct(array $data = [])
-    {
-        if (empty($data)) {
-            return $this;
-        }
-
-        $this->setPropertyList($data);
-
-        return $this;
-    }
-    #endregion
-
     #region [Public methods]
 
     /**
-     * @return array
+     * @return string
```

Comment:
Why did you change return type to string?

```suggestion
     * @return array
```

Reply:
It is for getType() method